### ISSUE TITLE
EF2 should be chattier while fitting ellipsoids

### ISSUE THREAD

**Issue by mdoube** [bonej-org/BoneJ2#211]
**Describe the bug**
Ellipsoid Factor 2 tells the user how many ellipsoids it is going to fit, but is then totally silent about what it is doing. It should minimally display progress and ideally log to console or the log window or the status bar how many ellipsoids are complete and how many remain.

Users may wonder if anything is happening and quit EF2 in frustration.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open an image and run _Plugins > BoneJ > Ellipsoid Factor 2_
2. Notice that there is no running commentary on progress

**Expected behavior**
Some kind of user feedback so that users can guess about how long they will be waiting for the ellipsoid optimisation step to complete.

**Additional context**
EF1 was super chatty, partly due to its prototype nature, with lots of lines like this in the log:
```
Ellipsoid at (3.8582000385820003, 1.61600001616, 3.8581999999999996) is invalid, nullifying after 7 iterations
```

**Comment by rimadoma** [comment]
@alessandrofelder You can use a `StatusService` to show messages and progress on the status bar. Messages with `statusService.showStatus("Confabulating tensor underflow")` and progress with `statusService.showProgress(completed, total)`.

**Comment by mdoube** [comment]
This is working now: sensible user feedback on the status bar is included in #209 

A progress bar indicator would be nice but will wait for the next release.
